src/modules/pivotTable/PivotTableEngine.js

```diff
@@ -744,7 +749,15 @@ export class PivotTableEngine {
                 totalCell.valueType = currentValueType
             }
 
-            if (dxDimension?.valueType === VALUE_TYPE_NUMBER) {
+            // compute subtotals and totals for all numeric and boolean value types
```

Comment:
I had to keep the part above with the prev/current comparison because `valueType` can be of types other than numeric and boolean.
In that case, the total cell would not get the type, but always N/A if we simply check for numeric/boolean for assigning the last evaluated `valueType` in the loop and fallback to N/A otherwise.